kong/templates/nginx_kong.lua

```diff
@@ -103,7 +103,7 @@ server {
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Proto $scheme;
-        proxy_set_header Host $upstream_host;
+        proxy_set_header Host $upstream_host:$upstream_port;
```

Comment:
Especially since it is not required for the port to be present if it is the default one for the service being requested, and we would have more control over that from the Lua land if we wish to perform such changes.